agent/nmap_options.py

```diff
@@ -97,6 +98,8 @@ def _set_ports_option(self) -> List[str]:
         """Appends the ports option to the list of nmap options."""
         if self.fast_mode is True:
             return ["-F"]
+        elif self.top_ports is not None:
+            return ["--top-ports", str(self.top_ports)]
```

Comment:
if top_ports has the same syntax as ports, ["--top-ports", self.top_ports] would work no?